examples/run_IR.py

```diff
@@ -31,6 +37,20 @@
     ph1 = 0.0
     ph2 = 0.0
     nPhaseSteps = 1
+# }}}
+# {{{ Parameters for Bridge12
+power = r_[config_dict["max_power"]]
+min_dBm_step = 0.5
+for x in range(len(power)):
+    dB_setting = (
+        round(10 * (log10(power[x]) + 3.0) / min_dBm_step) * min_dBm_step
+    )  # round to nearest min_dBm_step
+print("dB_setting", dB_setting)
+print("correspond to power in Watts", 10 ** (dB_setting / 10.0 - 3))
+input("Look ok?")
+power = 1e-3 * 10 ** (dB_setting / 10.0)
+# }}}
+
```

Comment:
calls from config file to set the power for the IR (so that IR can be run at powers if so desired